src/test/java/com/google/step/TestConstants.java

```diff
@@ -30,14 +30,8 @@
   public static final String USERNAME_B = "Bob";
   public static final String USERNAME_C = "Charlie";
 
-  public static final String BLOBKEY_A = "a_blob_key";
-  public static final String BLOBKEY_B = "a_blob_key_b";
-  public static final String BLOBKEY_C = "a_blob_key_c";
-
-  private static final String URL_PREFIX = "/api/images/";
-  public static final String BLOBKEY_URL_A = URL_PREFIX + BLOBKEY_A;
-  public static final String BLOBKEY_URL_B = URL_PREFIX + BLOBKEY_B;
-  public static final String BLOBKEY_URL_C = URL_PREFIX + BLOBKEY_C;
+  public static final String IMAGE_URL_A = "/api/images/" + BLOBKEY_A;
+  public static final String IMAGE_URL_B = "/api/images/" + BLOBKEY_B;
```

Comment:
This diff seems off to me - is BLOBKEY_A deleted now?

Can you please merge from master?

Reply:
Merged